layers/gpu/instrumentation/gpuav_shader_instrumentor.cpp

```diff
@@ -1010,83 +1013,98 @@ void GpuShaderInstrumentor::PostCallRecordPipelineCreationShaderInstrumentation(
     }
 }
 
-// While have an almost duplicated funciton is not ideal, the core issue is we have a single, templated function designed for
+// While have an almost duplicated function is not ideal, the core issue is we have a single, templated function designed for
 // Graphics, Compute, and Ray Tracing. GPL is only for graphics, so we end up needing this "side code path" for graphics only and it
 // doesn't fit in the "all pipeline" templated flow.
-void GpuShaderInstrumentor::PreCallRecordPipelineCreationShaderInstrumentationGPL(
-    const VkAllocationCallbacks *pAllocator, vvl::Pipeline &pipeline_state, vku::safe_VkGraphicsPipelineCreateInfo &new_pipeline_ci,
-    const Location &loc, std::vector<chassis::ShaderInstrumentationMetadata> &shader_instrumentation_metadata) {
+bool GpuShaderInstrumentor::PreCallRecordPipelineCreationShaderInstrumentationGPL(
+    const VkAllocationCallbacks *pAllocator, vvl::Pipeline &pipeline_state,
+    vku::safe_VkGraphicsPipelineCreateInfo &to_instrument_pipeline_ci, const Location &loc,
+    std::vector<chassis::ShaderInstrumentationMetadata> &shader_instrumentation_metadata) {
     // Init here instead of in chassis so we don't pay cost when GPU-AV is not used
     const size_t total_stages = pipeline_state.stage_states.size();
     shader_instrumentation_metadata.resize(total_stages);
 
     InstrumentationDescriptorSetLayouts instrumentation_dsl;
     BuildDescriptorSetLayoutInfo(pipeline_state, instrumentation_dsl);
 
-    auto library_create_info = const_cast<VkPipelineLibraryCreateInfoKHR *>(
-        vku::FindStructInPNextChain<VkPipelineLibraryCreateInfoKHR>(new_pipeline_ci.pNext));
+    auto to_instrument_pipeline_lib_ci = const_cast<VkPipelineLibraryCreateInfoKHR *>(
```

Comment:
for consistency, we have used `modified_` (ex `modified_create_info`) so rather we just do that then have some that are `to_instrument` and some that are `modified_`

... also we "instrument SPIR-V" we don't instrument a pNext struct

```suggestion
    auto modified_pipeline_library_ci = const_cast<VkPipelineLibraryCreateInfoKHR *>(
```

Reply:
ok!